source/Messaging/source/Messaging.Tests/ArchitectureTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Energinet.DataHub.Core.Messaging.Communication;
+using Energinet.DataHub.Core.Messaging.Communication.Publisher;
+using Energinet.DataHub.Core.Messaging.Communication.Subscriber;
+using FluentAssertions;
+using Google.Protobuf.Reflection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using Xunit.Categories;
+
+namespace Energinet.DataHub.Core.Messaging.Tests;
+
+[UnitTest]
+public class ArchitectureTests
```

Comment:
I don't see this as architecture. How about e.g.
```suggestion
public class ServiceRegistrationTests
```

Reply:
Renamed test class